grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx

```diff
@@ -32,481 +32,613 @@ import { IntegrationHelper } from 'pages/integration/Integration.helper';
 import { AppFeature } from 'state/features';
 import { useStore } from 'state/useStore';
 import { UserActions } from 'utils/authorization/authorization';
-import { PLUGIN_ROOT } from 'utils/consts';
-import { openErrorNotification } from 'utils/utils';
+import { PLUGIN_ROOT, generateAssignToTeamInputDescription } from 'utils/consts';
 
-import { getForm } from './IntegrationForm.config';
+import { HowTheIntegrationWorks } from './HowTheIntegrationWorks';
 import { prepareForEdit } from './IntegrationForm.helpers';
-import styles from './IntegrationForm.module.scss';
+import { getIntegrationFormStyles } from './IntegrationForm.styles';
+
+enum FormFieldKeys {
```

Comment:
I don't see an advantage of having this enum instead of passing keys directly to `FormFields` interface. In all the places we have intellisense & type safety when using keys directly. Not a big issue but I just think it's unnecessary in this case. Wdyt?

Reply:
Refactoring is a good reason! Let's leave it then